website/docs/faq.md

```diff
@@ -252,20 +252,45 @@ bundle exec srb rbi suggest-typed
 
 ## What platforms does Sorbet support?
 
-The `sorbet` and `sorbet-runtime` gems are currently only tested on Ruby 2.5 and
-Ruby 2.6.
-
-Ruby 2.7 has
-[known issues](https://github.com/sorbet/sorbet/issues?q=is%3Aissue+2.7+) and is
-[not being worked on yet](https://github.com/sorbet/sorbet/issues/2771#issuecomment-599761098)
-(as of May 2020) but PRs are welcome!
-
-The static checker is only tested on macOS 10.14 (Mojave) and Ubuntu 18 (Bionic
-Beaver). We expect it to work on macOS 10.10 (Yosemite) and most Linux
-distributions where `glibc`, `git` and `bash` are present.
-
-If you are using one of the official minimal Ruby Docker images you will need to
-install the extra dependencies yourself:
+The `sorbet-runtime` gem is currently only tested on Ruby 2.5 and Ruby 2.6, and
+we believe it to work on Ruby 2.7. It is known to not support Ruby 2.4. Feel
+free to report issues for any current or future Ruby version.
+
+Feel free to report runtime issues for any current or future Ruby version.
+
+The `sorbet-static` gem aims to support a superset of all Ruby versions. In
+particular, the standard library [RBI files](rbi.md) might reflect classes,
```

Comment:
```suggestion
We publish a single `sorbet-static` gem that supports all Ruby versions, rather than
supplying different `sorbet-static` gems for each Ruby version. This means that the standard
library [RBI files](rbi.md) might reflect classes,
```

Reply:
> We publish a single `sorbet-static` gem that supports all Ruby versions, rather than

We actually publish like 10 different sorbet-static gems, technically. Do you feel strongly about your wording? I worry that it's incorrect, technically speaking.

Reply:
Eh, yeah, don't say it if it's incorrect. :)

Reply:
I think the only reason I was poking on this is that I found the phrasing `superset of all Ruby versions` a bit confusing. Maybe something like `the union of all Ruby versions` or... I dunno.